lib/player.rb

```diff
@@ -0,0 +1,12 @@
+class Player
+  attr_reader :mark
+  def initialize(mark)
+    # X or O
+    @mark = mark
+  end
+
+  def get_input(display)
```

Comment:
When we think about display (things logged to the console), can we place them in a class to encapsulate that particular responsibility?

Reply:
Ah, I see what you're doing, ie passing the display in places. Maybe instead of magic strings in various files, you can group them in one constants messages file and then access them when needed.